restcomm/restcomm.notification/src/main/java/org/restcomm/connect/notification/GlobalNotification.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.restcomm.connect.notification;
+
+import java.net.URI;
+import org.apache.commons.configuration.Configuration;
+import org.joda.time.DateTime;
+import org.restcomm.connect.dao.DaoManager;
+import org.restcomm.connect.dao.NotificationsDao;
+import org.restcomm.connect.dao.entities.Notification;
+import org.restcomm.connect.commons.dao.Sid;
+
+/**
+ *
+ * @author charles.roufay
+ */
+public class GlobalNotification {
```

Comment:
The GlobalNotification should be an Akka actor that the rest of the actors should use to send notification to be created.
For example, from CallManager we should have notificationService.tell("notification blabla", self());
Also maybe provide a Notification API so we can have different implementations of the Notification service, for example one implementation would only generate Notification another implementation would also send SMS for certain types of Notifications.